Makefile

```diff
@@ -48,14 +48,16 @@ ALL_CPPFLAGS	+= -D_GNU_SOURCE
 ALL_CPPFLAGS	+= $(shell pkg-config --cflags libzip)
 ALL_CPPFLAGS	+= -I/usr/include
 ALL_CPPFLAGS	+= -I/usr/local/include
-ALL_CPPFLAGS	+= -I/usr/local/opt/curl/include
-ALL_CPPFLAGS	+= -I/usr/local/opt/libzip/include
```

Comment:
please dont remove the `/usr/local/opt` dirs, they are still used by Intel Homebrew

Reply:
Yeah, that was accidental. Reverted, and opened https://github.com/tldr-pages/tldr-c-client/issues/80 to track fixing this.